src/components/TodoList/TodoList.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable react/prop-types */
+import React from 'react';
+import './TodoList.css';
+
+function TodoList({
+  // eslint-disable-next-line react/prop-types
+  todos, setTodos,
+}) {
+  const updateTask = (id) => {
+    // eslint-disable-next-line react/prop-types
```

Comment:
Please don't remove linters, look for a way so that you can make the exercise without linters skipping.
```suggestion
import React from 'react';
import './TodoList.css';

function TodoList({
  todos, setTodos,
}) {
  const updateTask = (id) => {
```